src/main/kotlin/no/nav/familie/ks/sak/common/util/StringListConverter.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.familie.ks.sak.common.util
+
+import javax.persistence.AttributeConverter
+import javax.persistence.Converter
+
+@Converter
+class StringListConverter : AttributeConverter<List<String>, String> {
+
+    override fun convertToDatabaseColumn(stringList: List<String>): String {
+        return java.lang.String.join(SPLIT_CHAR, stringList)
+    }
```

Comment:
Hmm, funker ikke kotlin sin string join?
Noe ala

```suggestion
    override fun convertToDatabaseColumn(stringList: List<String>): String =
        stringList.joinToString(separator = SPLIT_CHAR)
```